Modules/ModParser-3_0.lua

```diff
@@ -611,6 +611,7 @@ local modFlagList = {
 	["with channelling skills"] = { tag = { type = "SkillType", skillType = SkillType.Channelled } },
 	["channelling skills"] = { tag = { type = "SkillType", skillType = SkillType.Channelled } },
 	["with brand skills"] = { tag = { type = "SkillType", skillType = SkillType.Brand } },
+	["for stance skills"] = { tag = { type = "SkillName", skillName = "Blood and Sand" } },
```

Comment:
You forgot Flesh and Stone, it's also a Stance skill.

Reply:
Whoops, forgot about that one